HTTP Server.core/Server.cs

```diff
@@ -82,18 +85,40 @@ private  byte[] HandleReply(byte[] request, int requestSize)
             }
             catch
             {
-                return Encoding.UTF8.GetBytes("HTTP/1.1 400 Bad Request\r\n");
+                return Encoding.UTF8.GetBytes(CreateMessage("HTTP/1.1 400 Bad Request\r\n"));
             }
-            if (version.Equals("HTTP/1.1\r\n") && isRequestMethod(method))
+            if (version.Equals("HTTP/1.1\r\n") && IsRequestMethod(method))
             {
                 if (uri.Equals("/"))
-                    return Encoding.UTF8.GetBytes("HTTP/1.1 200 OK\r\n");
-                return Encoding.UTF8.GetBytes("HTTP/1.1 404 Not Found\r\n");
+                    return Encoding.UTF8.GetBytes(CreateMessage("HTTP/1.1 200 OK\r\n"));
+                return Encoding.UTF8.GetBytes(CreateMessage("HTTP/1.1 404 Not Found\r\n"));
             }
-            return Encoding.UTF8.GetBytes("HTTP/1.1 400 Bad Request\r\n");
+            return Encoding.UTF8.GetBytes(CreateMessage("HTTP/1.1 400 Bad Request\r\n"));
         }
 
-        private bool isRequestMethod(string str)
+        public string CreateMessage(string str)
```

Comment:
Once you're adding a body you need to indicate how many bytes are in it using the `Content-Length` header.